x-pack/plugins/cloud_security_posture/server/routes/csp_rule_template/get_csp_rule_template.ts

```diff
@@ -23,6 +25,21 @@ import {
 import { CspRouter } from '../../types';
 import { PACKAGE_POLICY_SAVED_OBJECT_TYPE } from '../benchmarks/benchmarks';
 
+export const getSortedCspRulesTemplates = (cspRulesTemplates: CspRuleTemplate[]) => {
+  return cspRulesTemplates.slice().sort((a, b) => {
+    const versionA = semverValid(a?.metadata?.benchmark?.rule_number);
+    const versionB = semverValid(b?.metadata?.benchmark?.rule_number);
+
+    if (versionA !== null && versionB !== null) {
+      return semverCompare(versionA, versionB);
+    } else {
+      return String(a?.metadata?.benchmark?.rule_number).localeCompare(
```

Comment:
nit: move `a?.metadata?.benchmark?.rule_number` and `b?.metadata?.benchmark?.rule_number` to a const